docs/Resampling_Benchmarks.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>While benchmarking cuda code, you need to call <code>torch.cuda.synchronize()</code> so it waits until the cuda computation is over before returning, as the calls are asyncronous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastaudio/fastaudio/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>